kythe/cxx/verifier/souffle_interpreter.cc

```diff
@@ -190,10 +211,13 @@ SouffleResult RunSouffle(const SymbolTable& symbol_table,
   souffle::IOSystem::getInstance().registerWriteStreamFactory(
       write_stream_factory);
   souffle::IOSystem::getInstance().registerReadStreamFactory(
-      std::make_shared<KytheReadStreamFactory>(anchors, database));
+      std::make_shared<KytheReadStreamFactory>(anchors, database,
+                                               symbol_table));
   auto ram_tu = souffle::ParseTransform(absl::StrCat(
       program.code(), ".output result(IO=kythe, id=", output_id, ")\n"));
   if (ram_tu == nullptr) {
+    fprintf(stderr, "no ram_tu for program <%s>\n",
```

Comment:
just LOG(...) and then you can get rid of the awkward string_view -> string -> `const char*` shenanigans